docs/revsets.md

```diff
@@ -216,8 +216,10 @@ revsets (expressions) as arguments.
   change is divergent, this resolves to multiple commits. It is an error to use a
   non-unique prefix. Unmatched prefix isn't an error.
 
-* `commit_id(prefix)`: Commits with the given commit ID prefix. It is an error
-  to use a non-unique prefix. Unmatched prefix isn't an error.
+* `commit_id(prefix)`: Commit with the given commit ID prefix. It is an error
```

Comment:
nit: I don't think changing the Commits to Commit is correct here, since there may be multiple with the same prefix. That's taken from https://github.com/jj-vcs/jj/pull/6912#issuecomment-3050911051

Reply:
As it says right below, "It is an error to use a non-unique prefix".

I think we, potentially, might also want something like `non_unique_commit_id` that doesn't have that restriction (with a better name), but it'd be used much less frequently. In my mind, it'd be used mainly to resolve a commit id among a custom set of commits, e.g. (using imaginary functions) `non_unique_commit_id(abc) & today()`.